outbound/hmail.js

```diff
@@ -966,8 +986,22 @@ class HMailItem extends events.EventEmitter {
             }
         });
 
+        const escaped_chars = {
+            "&": "&amp;",
+            "<": "&lt;",
+            ">": "&gt;"
+        };
+
```

Comment:
I'd suggest going a bit further:

```javascript
	const escaped_chars = {
		'<': 'lt',
		'>': 'gt',
		'"': 'quot',
		'\'': 'apos',
		'&': 'amp',
		'\r': '#10',
		'\n': '#13'
	};
```
(I left out the `&` and `;` since you can add them once in the replacement).

Reply:
I can't imagine why anyone would use variables for tag parameters and need escaping quotes or apostrophes but I see no problem with that. I am slightly opposed with escaping \r\n - itself it don't break anything, even after escaping pre tag is required to properly view lines...

Reply:
...or can it break email itself? Something like \r\n\r\n.\r\n?